### PR TITLE
Cronjob Job Limit

### PR DESCRIPTION
**Pull request by carlostrr**
## Description
Changes:
- Add Job history fields to scan CRD;
- Pass scan Job history amount to the Scan Cronjob;

## How has this been tested?
With local deployments on a virtual cluster and go-test.

## Checklist
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/getupio-undistro/.github/labels?q=Type%3A)
- [x] I have documented my code (if applicable)
- [x] My changes are covered by tests

### CODE REVIEW

File: apis/zora/v1alpha1/clusterscan_types.go [hunk 1]
```diff
@@ -39,6 +39,18 @@ type ClusterScanSpec struct {
 
 	// The list of Plugin references that are used to scan the referenced Cluster.  Defaults to 'popeye'
 	Plugins []PluginReference `json:"plugins,omitempty"`
+
+	// SuccessfulScansHistoryLimit specifies the amount of successfully
+	// completed scan Jobs to be kept in the cluster. This field is analogous
+	// to <Cronjob.Spec.SuccessfulJobsHistoryLimit> from the <batch> package.
+	// +kubebuilder:validation:Minimum=0
+	SuccessfulScansHistoryLimit *int32 `json:"successfulScansHistoryLimit,omitempty"`
```
Comment (matheusfm) [inline apis/zora/v1alpha1/clusterscan_types.go:47]: We need a default value for this field, so I recommend add this marker: `// +kubebuilder:default=3`

File: apis/zora/v1alpha1/clusterscan_types.go [hunk 2]
```diff
@@ -39,6 +39,18 @@ type ClusterScanSpec struct {
...
+
+	// FailedScansHistoryLimit specifies the amount of failed scan Jobs to be
+	// kept in the cluster. This field is analogous to
+	// <Cronjob.Spec.FailedScansHistoryLimit> from the <batch> package.
+	// +kubebuilder:validation:Minimum=0
+	FailedScansHistoryLimit *int32 `json:"failedScansHistoryLimit,omitempty"`
```
Comment (matheusfm) [inline apis/zora/v1alpha1/clusterscan_types.go:53]: We need a default value for this field, so I recommend add this marker: `// +kubebuilder:default=1`

**Review by matheusfm — CHANGES_REQUESTED** [review]

File: apis/zora/v1alpha1/clusterscan_types.go (hunk 1 above)
Comment (carlostrr) [reply apis/zora/v1alpha1/clusterscan_types.go:48]: Done

File: apis/zora/v1alpha1/clusterscan_types.go (hunk 2 above)
Comment (carlostrr) [reply apis/zora/v1alpha1/clusterscan_types.go:55]: Done

**Review by matheusfm — APPROVED** [review]

**Pull request merged**